### ISSUE TITLE
Add "Equipment" to the possible list of media

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2WebSDL#74]
Also change all existing "board temp" and "battery voltage under load" instance to have a media type of equipment.

**Comment by horsburgh** [comment]
@jcaraballo17 - these are simple updates in the database. I'm assigning to you.

**Comment by horsburgh** [comment]
@SRGDamia1 - we aren't sure what "board temp" and "battery voltage under load" are. I'm assuming these are variables? @jcaraballo17  is having trouble matching these up with something in the database. Can you be more specific for him?

**Comment by jcaraballo17** [comment]
BUMP. 
I added the Equipment medium, but I couldn't figure out which ones are the board temp and the battery voltage under load variables.

**Comment by SRGDamia1** [comment]
Sorry!  Variables that should be "equipment:"
- Temperature (EnviroDIY_Mayfly_Temp) (board temp)
-  Battery voltage (EnviroDIY_Mayfly_Volt) (battery voltage under load)
- Free SRAM (EnviroDIY_Mayfly_FreeSRAM)

**Comment by horsburgh** [comment]
This one looks good on the sandbox. I think it can be closed when deployed to production.

**Comment by aufdenkampe** [comment]
I'm happy to close now, as it will be resolved when pushed to production.

**Comment by aufdenkampe** [comment]
This looks good on Sandbox, but I'm a bit concerned that we're overloading the concept of "Variable" in our building of this EnviroDIY system and the underlying ODM2 database. The EnviroDIY deployment of ODM2 will showcase a lot of ODM2 capabilities, and it'll be used by many others. I fear that we'll have problems as we develop future applications and demonstrations on top of what we have.

The list that Sara provided (EnviroDIY_Mayfly_Temp; EnviroDIY_Mayfly_Volt; EnviroDIY_Mayfly_FreeSRAM) are labels for sub types of Results, these are not Variables.

In ODM2 "Medium" is an attribute of a Result, "Variable" is the attribute of a Result, and "Equipment" Make & Model are attributes of an Action (which is always tied to a Result in the case of Observation Actions). See http://odm2.github.io/ODM2/schemas/ODM2_Current/diagrams/ODM2OverviewSimplified.html and http://odm2.github.io/ODM2/schemas/ODM2_Current/diagrams/ODM2Equipment.html.

Perhaps we can resolve this comment when we tackle issue #71 (Allow super-users to add new variables and sensors), which is scheduled for https://github.com/ODM2/ODM2WebSDL/milestone/5.

**Comment by aufdenkampe** [comment]
Reopened just to get thoughts on my last comment.

**Comment by horsburgh** [comment]
@aufdenkampe - "EnviroDIY_MayFly_Temp" is a concatenation of Sensor make and model and the variable. It's a convenient identifier of sorts for these combinations. These concepts are separate in the underlying database. We don't have to display them this way. But - we have made some convenient simplifications that I believe make things easier to use for people who don't have (and don't want) an intimate knowledge of the underlying database structure. 

I'm not sure what the remaining issue is here?

We decided a long time ago that the measured variables were going to be closely tied to the sensor.
